src/reporters/tiny.js

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+
+export default report;
+
+async function report(runningTests, startTime) {
+  console.log("");
+  const summary = await reportSequentially(runningTests);
+  const { failed, passed, endTime } = summary;
+  const ok = color.green(`✓ ${passed}`);
+  const notOk = failed ? color.red(`✗ ${failed}`) : "";
```

Comment:
I personally I prefer using `failed > 0` because it shows `failed` is a number (integer even).

Alternatively change the name to `failedCount` or something similar. Either way helps maintainability/readability.

Reply:
You're right, `failed` sounds like a boolean instead of an integer.